recipes/newrelic/postgres/debian.yml

```diff
@@ -0,0 +1,148 @@
+name: postgres-open-source-integration
+displayName: Postgres Open Source Integration
+description: New Relic install recipe for default Postgres Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-postgresql
+
+installTargets:
+  - type: host
+    os: linux
+    platform: "debian"
+  - type: host
+    os: linux
+    platform: "ubuntu"
+
+# keyword convention for dealing with search terms that could land someone on this instrumentation project
+keywords:
+  - Infrastructure
+  - Integration
+  - postgres
+
+# CLI runs process detection; this is used to filter recipes that are appropriate for matched processes
+processMatch:
+  - postgres
+
+# Matches partial list of the Log forwarding parameters
+logMatch:
+  - name: postgresql
+    file: /var/log/postgresql/postgresql*.log
+
+validationNrql: "SELECT count(*) from PostgresqlDatabaseSample where hostname like '{{.HOSTNAME}}' FACET entityGuid SINCE 10 minutes ago"
+
+inputVars:
+  - name: "NR_CLI_USERNAME"
+    prompt: "Postgres Username"
+  - name: "NR_CLI_PASSWORD"
+    prompt: "Postgres Password"
+  - name: "NR_CLI_DATABASE"
+    prompt: "Postgres Database (default: postgres)"
+    default: "postgres"
+  - name: "NR_CLI_PORT"
+    prompt: "Postgres Port (default: 5432)"
+    default: 5432
+  - name: "NR_CLI_SSL"
+    prompt: "Use SSL to connect? (true/false)"
+  - name: "NR_CLI_CERT_AUTH_FILE"
+    prompt: "Certificate Authority file (if applicable)"
+  - name: "NR_CLI_TRUST_SERVER_CERTIFICATE"
+    prompt: "Trust certificate? (true/false)"
+  - name: "NR_CLI_CLIENT_CERT_FILE"
+    prompt: "Client Certificate file (if applicable)"
+  - name: "NR_CLI_CERT_KEY"
+    prompt: "PEM key file (if applicable)"
```

Comment:
Need to do conditional logic based off the values for `NR_CLI_SSL` and `NR_CLI_TRUST_SERVER_CERTIFICATE` before writing to `postgresql-config.yml`. Details are outlined in #66.